projects/packages/jetpack-mu-wpcom/package.json

```diff
@@ -21,7 +21,8 @@
 		"build-production-js": "NODE_ENV=production BABEL_ENV=production pnpm build-js",
 		"lint": "eslint --ext .js,.jsx,.cjs,.mjs,.ts,.tsx .",
 		"clean": "rm -rf src/build/",
-		"watch": "pnpm run build-production-js && pnpm webpack watch"
+		"watch": "pnpm run build-production-js && pnpm webpack watch",
+		"e2e-tests": "playwright test --config src/features/verbum-comments/playwright.config.ts"
```

Comment:
Im getting a "No tests found" error when I run `pnpm run e2e-tests` from the jetpack-mu-wpcom root folder.

Reply:
<img width="915" alt="Screenshot 2024-01-31 at 10 00 46" src="https://github.com/Automattic/jetpack/assets/7000684/51ff71f2-6da5-4405-95e5-d1ba644455d0">
Its working for me.
Can you double-check the folder and node version?